qml/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+	DESTINATION ${DATADIR}
+	PATTERN "platform*" EXCLUDE)
+install(DIRECTORY platform.generic
+	DESTINATION ${DATADIR}/qml)
+
+if(FLAVOR STREQUAL "kirigami")
+	install(DIRECTORY platform.kirigami/
+		DESTINATION ${DATADIR}/qml/platform)
+	install(DIRECTORY platform.qtcontrols/
+		DESTINATION ${DATADIR}/qml/platform)
```

Comment:
This fails and creates non-functioning install. Instead, we have to install platform using 

```
cp -L -v $$PWD/qml/platform.$$FLAVOR/*.qml ${INSTALL_ROOT}$$DATADIR/qml/platform
```

or similar in cmake. What happened here was that some kirigami files got overwritten with qtcontrols and Pure Maps was unable to open a menu. 

The symlinks between platforms are decided while developing and it will be nasty location for errors if they happen on install. So, we should drop `platform.generic` installation above and use either some `cp` command or internal cmake module to dereference links on installation.

As FLAVOR is a part of the platform name, we also don't need `if ... elseif...` construction, but can use the var immediately, as it was done in QMAKE PRO file.

https://github.com/rinigus/pure-maps/blob/master/pure-maps.pro#L158